### ISSUE TITLE
Add switches for SC2034 and SC2154

### ISSUE THREAD

**Issue by skbeh** [claui/vscode-aur-packaging#19]
There are people (like me) who want to occasionally check these warnings, so it would be better to let users decide it.
Also, please remove the unused `Executable Path`.

**Comment by claui** [comment]
> please remove the unused Executable Path.

Addressed in #22, thanks for your feedback @skbeh!

> There are people (like me) who want to occasionally check these warnings, so it would be better to let users decide it.

Have you tried adding the following to your VS Code settings?

```jsonc
{
  "[aur-pkgbuild]": {
    "shellcheck.exclude": [
      // "2034",
      // "2154",
      "2164"
    ]
  }
}
```

That should override the defaults to your liking. It should work both in user and workspace settings.  
Let me know if that helps.

**Comment by claui** [comment]
I just double-checked that this override actually works.

Closing for now. Feel free to follow up regardless.
